File: benchmarks/solutions/swift/a1-two-sum.swift
func twoSum(_ nums: [Int], _ target: Int) -> (Int, Int) {
    var seen: [Int: Int] = [:]
    for (i, n) in nums.enumerated() {
        let complement = target - n
        if let j = seen[complement] {
            return (j, i)
        }
        seen[n] = i
    }
    fatalError("no solution")
}

File: benchmarks/solutions/csharp/a1-two-sum.cs
using System.Collections.Generic;

public class Solution
{
    public static (int, int) TwoSum(int[] nums, int target)
    {
        var seen = new Dictionary<int, int>();
        for (int i = 0; i < nums.Length; i++)
        {
            int complement = target - nums[i];
            if (seen.TryGetValue(complement, out int j))
                return (j, i);
            seen[nums[i]] = i;
        }
        return (-1, -1);
    }
}
